From 1a8744d733ec64126961f76e4e5b221b57a7b485 Mon Sep 17 00:00:00 2001
From: Etaash Mathamsetty
 <45927311+Etaash-mathamsetty@users.noreply.github.com>
Date: Sun, 25 May 2025 22:22:17 -0400
Subject: [PATCH 067/166] winewayland: Add error message when
 wp_color_manager_v1 is missing.

---
 dlls/winewayland.drv/wayland.c | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/dlls/winewayland.drv/wayland.c b/dlls/winewayland.drv/wayland.c
index fc0665309a5..29f3e48645e 100644
--- a/dlls/winewayland.drv/wayland.c
+++ b/dlls/winewayland.drv/wayland.c
@@ -346,6 +346,9 @@ BOOL wayland_process_init(void)
     if (!process_wayland.wp_fractional_scale_manager_v1)
         ERR("Wayland compositor doesn't support wp_fractional_scale_manager_v1 (fractional scaling will be broken)\n");
 
+    if (!process_wayland.wp_color_manager_v1)
+        ERR("Wayland compositor doesn't support wp_color_manager_v1 (HDR will not be supported)\n");
+
     process_wayland.initialized = TRUE;
 
     return TRUE;
-- 
2.50.0

